grammars/javascript.cson

```diff
@@ -300,6 +300,29 @@
   }
   {
     'name': 'meta.function.arrow.js'
+    'begin': '([a-zA-Z_?$][\\w?$]*)\\s*(=)\\s*(\\()(?=(?:[^\\(\\)]*)?\\)\\s*=>)'
+    'beginCaptures':
+      '1':
+        'name': 'entity.name.function.js'
+      '2':
+        'name': 'keyword.operator.js'
+      '3':
+        'name': 'punctuation.definition.parameters.begin.js'
+    'comment': 'match stuff like: play = (a,b) => { … }'
```

Comment:
This comment should probably either go to the top or bottom of the rule (I realize the comments in this file have no consistency at all though, just starting to try to clean them up :P)